src/style.css

```
@@ -1 +1,72 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: var(--default-font-weight);
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin-top: auto;
+}
+
+.form-input {
+  display: block;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-weight: var(--default-font-weight);
+  background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  appearance: none;
+  outline: none; /* прибераємо обводку */
```

Remove comments everywhere
```suggestion
  outline: none; 
```